src/os/command/feature/featureiconcmd.js

```diff
@@ -54,16 +53,4 @@ export default class FeatureIcon extends AbstractFeatureStyle {
 
     super.applyValue(configs, value);
   }
-
-  /**
-   * @inheritDoc
-   */
-  finish(configs) {
```

Comment:
Do we need to dispatch some other event for icon changes? Just want to be sure the old event wasn't papering over some other need to notify an icon change.

Reply:
The old comment was copy/pasted from the layer level command. This dispatch was only handled by `KMLNode`, but was lumped in with a color change which did more than needed. A color change needs to propagate the `icons` event up the tree so the node icon color will be updated. An icon change only needs to fire an event to trigger saving the KML (for Saved Places), which is what the style event is doing in the abstract command.